android/java/org/chromium/chrome/browser/widget/quickactionsearchandbookmark/QuickActionSearchAndBookmarkWidgetProvider.java

```diff
@@ -93,12 +93,11 @@ public void accept(SearchActivityPreferences prefs) {
 
     public static String FROM_SETTINGS = "FROM_SETTINGS";
 
-    private static final int TOTAL_TILES = 16;
+    private static final int TOTAL_TILES = 12;
```

Comment:
We support only 12 top tiles at the moment(4 per row, so 3 rows in total), it could be we had 4, but upstream saves only 12 right now.